### Investigate null values in item_type_code_name

In [2]:
%run '../lib/libraries.ipynb'

dython              0.6.1
tinydb              4.2.0


In [3]:
dfFlt = load_df('dfFlt')

###### Create a dataframe for investigation

In [4]:
nullItDf = dfFlt.filter(F.col('item_type_code_name').isNull())

###### Number of null values

In [5]:
nullItDf.select().count()

3264

- 3264
- close to the 3298 nulls in response correctness


###### Display any single values

In [6]:
cols = single_val(nullItDf.toPandas())
display_sv_cols(nullItDf, cols)

0
assignment_max_attempts                    1
assignment_late_submission             False
assignment_attempt_number                  1
is_manual_scoring_required             False
item_is_offline_scored                 False
learner_attempt_status          fully scored
item_type_code_name                     None
scoring_type_code               [unassigned]
response_correctness                    None
assigned_item_status                assigned
is_deleted                             False
assessment_instance_attempt_id           NaN
assessment_item_response_id              NaN

- Looks like it wasn't answered
  - response_correctness = Null (None)
  - is a subset of the 3298 nulls in response correctness

- Will be resolved when the 3298 null in response_correctness is resolved

In [16]:
finish_todo('Investigate null values in item_type_code_name')